### PR TITLE
Implementing Tourguide for issue #28

### PR DESCRIPTION
**Pull request by Chukslord1**

### CODE REVIEW

**Comment by litenull** [comment]
This looks good in general.

Some comments:

1. Why do we need sass if we have styled components?
2. Put default values in variables on top instead of hard coding

**Comment by sutjin** [comment]
Overall looks good, a bit of a nitpick - would prefer if we use ID instead of class since we want to ensure the components going on tour is unique, but this one works too

File: web/src/HelpMode/TooltipHelp.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
+import React, { useState, useEffect } from "react";
```
Comment (sutjin) [inline web/src/HelpMode/TooltipHelp.js:1]: We should move this file into the `/components` directory

File: web/src/HelpMode/TooltipHelp.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    clearInterval(timeout);
+    setActive(false);
+  };
+
+  useEffect(() => {
+    setActive(show);
+  }, [show]);
+
+  return (
+    <div
```
Comment (sutjin) [inline web/src/HelpMode/TooltipHelp.js:27]: also noticed you built a tooltip by hand. we are using Chakra-ui and some of the components may already come built in with the library

**Comment by Chukslord1** [comment]
@litenull @sutjin  Please review. I think this pull request is ready for merging

**Pull request merged**